### PR TITLE
Add account activaiton endpoint

### PR DESCRIPTION
**Pull request by dosvath**
## Problem

No documentation on linking a bank account

## Solution

Add account activation endpoint.

## Testing Done

- [x] Spec builds using `redoc-cli serve`
- [x] Previewed changes locally

### CODE REVIEW

File: openapi/openapi.yml [hunk 1]
```diff
@@ -321,6 +331,65 @@ paths:
...
+                - account_holder_id
+              properties:
+                mentum_user_id:
+                  $ref: "#/components/schemas/mentum_user_id"
+                account_number:
+                  type: string
+                  description: The account number of the user's bank account.
+                  maxLength: 20
+                currency:
+                  $ref: "#/components/schemas/currency"
```
[the remark this answers is not in the document]
Comment (dosvath) [reply openapi/openapi.yml]: good find, updated

File: openapi/openapi.yml [hunk 2]
```diff
@@ -321,6 +331,68 @@ paths:
           $ref: "#/components/responses/NotAllowed"
         "500":
           $ref: "#/components/responses/InternalError"
+  /account/activate:
+    post:
+      tags:
+        - Account Opening
+      operationId: activateAccount
+      summary: Activate Account
+      description: Activate the user's account by linking a banks account to use for deposits.
```
Comment (ncassata) [inline openapi/openapi.yml:340]: Would this account also be used for withdrawals?

Comment (dosvath) [reply openapi/openapi.yml]: yes

File: openapi/openapi.yml [hunk 3]
```diff
@@ -321,6 +331,68 @@ paths:
...
+      operationId: activateAccount
+      summary: Activate Account
+      description: Activate the user's account by linking a banks account to use for deposits and withdrawals.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - mentum_user_is
```
[the remark this answers is not in the document]
Comment (dosvath) [reply openapi/openapi.yml]: great catch

**Pull request merged**